vulture/core.py

```diff
@@ -143,7 +145,12 @@ def __hash__(self):
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
 
-    def __init__(self, verbose=False):
+    def __init__(self, verbose=False, ignore_names=[]):
```

Comment:
[] -> None (http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)

and below:

`ignore_names = ignore_names or []`